jobs/azure_cpi/spec

```diff
@@ -19,45 +19,54 @@ properties:
     description: Azure region name
     example: eastus
   azure.subscription_id:
-    description: Azure Subscription Id
-  azure.resource_group_name:
-    description: Resource group name to use when spinning up new vms
-  azure.storage_account_name:
-    description: Azure storage account name. This property is required when use_managed_disks is false or upgrading from an existing deployment whose use_managed_disks is false.
+    description: Azure Subscription ID
+  azure.credentials_source:
+    description: Where to get Azure credentials for the azure cpi. This can be set to `static` to use `tenant_id`, `client_id` and `client_secret`/`certificate` or `managed_identity` to get the credentials from Azure managed service identity.
```

Comment:
can be confused, suggest:
Where to get Azure credentials for the azure cpi. This can be set to `static` to use `tenant_id`, `client_id` and `client_secret`/`certificate`, or set to  `managed_identity` to get the credentials from Azure managed service identity.